### ISSUE TITLE
Adding Repositories - Status 'OK', but not appearing on profile

### ISSUE THREAD

**Issue by MHillier98** [sourcerer-io/sourcerer-app#527]
I've attempted the process several times, but haven't had any success in actually adding any repositories to my profile.

If I use the Refresh button to update my GitHub repos, it gives the message "Finished 2 repos ok, 5 skipped."
I've also tried using the desktop app, which doesn't give me any errors and appears to process correctly?

Maybe it takes more time to add to my profile, or I'm misunderstanding something?

**Comment by yaronskaya** [comment]
Hi @MHillier98! Check your profile - the repositories have been added to your account.
FYI commits in your repositories had invalid emails. If you plan to process new repositories, consider changing wrong emails. See (changing author info)[https://help.github.com/en/articles/changing-author-info]

**Comment by MHillier98** [comment]
Thanks!

**Comment by gigashiro** [comment]
+1 having same problem here with my gitlab account. I've checked my email.

**Comment by yaronskaya** [comment]
Hi @gigashiro. Looks like you don't have any personal projects in your gitlab account.
By the way, you can process any repositories locally using the app.

**Comment by gigashiro** [comment]
Sorry, I just forgot this issue. @yaronskaya only personal projects are eligible? Cause I have a little bit more projects than none:
![image](https://user-images.githubusercontent.com/9381924/64896874-b9d8c200-d657-11e9-8713-c0a3484c6176.png)

**Comment by yaronskaya** [comment]
@gigashiro yes, currently only personal projects can be processed through the site. One way to add other projects is through the use of sourcerer-app.

**Comment by smaharj1** [comment]
I also have a problem with the emails. I have multiple projects in my GitHub but I recently added a second email (which I made my primary email). My sorcerer only shows 1 repo and I am not sure how to fix this since both of my emails are still connected to my Github. My profile: [https://sourcerer.io/smaharj1](https://sourcerer.io/smaharj1)

**Comment by Enrique1987** [comment]
I have the same problem, in "Refresh" I have 3 Repositories with status ok but appears nothing. 
Profil: https://sourcerer.io/enrique1987
could be that take long to show the code ? more than one week ?

**Comment by ujorgeleite** [comment]
Hey @MHillier98 and @Enrique1987 I'm with the same problem. How you fix your problem? my profile: https://sourcerer.io/ujorgeleite
